src/openrct2/rct12/ScenarioPatcher.cpp

```diff
@@ -56,6 +57,10 @@ static const std::string s_toKey = "to";
 static const std::string s_tilesKey = "tiles";
 static const std::string s_typeKey = "type";
 
+// Surface fix keys
+static const std::string s_surfacesKey = "surfaces";
+static const std::string s_destinationSurface = "to_surface";
```

Comment:
This does not adhere to our codestyle rules.
```suggestion
static const std::string _surfacesKey = "surfaces";
static const std::string _destinationSurface = "to_surface";
```

Reply:
Thanks for the reminder, I've tackled them all on the second commit.